applications/nrf_desktop/doc/board.rst

```diff
@@ -0,0 +1,39 @@
+.. _board:
+
+Board module
+############
+
+The ``board`` module is used to assure proper state of GPIO pins which are connected to electrical components and not configured by other modules in the application.
+
+Module Events
+*************
+
++----------------+-------------+--------------+-----------------+------------------+
+| Source Module  | Input Event | This Module  | Output Event    | Sink Module      |
++================+=============+==============+=================+==================+
+|                |             | ``board``    |                 |                  |
++----------------+-------------+--------------+-----------------+------------------+
+
+Configuration
+*************
+
+The module uses the Zephyr :ref:`zephyr:gpio` driver to set pin state.
+For this reason, you should set :option:`CONFIG_GPIO` option.
+
+For every configuration you must define ``port_state_def.h`` file (in board-specific directory in the application configuration folder).
+
+Mentioned file defines port states by the following arrays:
+
+* ``port_state_on`` - state set on system start and wakeup.
+* ``port_state_off`` - state set when system enters low-power mode.
+
+Every :c:type:`struct port_state` refers to a single GPIO port and contains information about:
+
+* :cpp:member:`name` - the GPIO device name (defined by symbol in :ref:`zephyr:device-tree`).
```

Comment:
```suggestion
* :cpp:member:`name` - GPIO device name (defined by a symbol in :ref:`zephyr:device-tree`).
```
What kind of symbol? Provide an example.

Reply:
ok